### DISCUSSION TITLE
security onion web code Where to download？  what language？

### DISCUSSION THREAD

**Discussion by wangfei906** [Security-Onion-Solutions/securityonion#8195] [General] [answered]
security onion web code Where to download？

**Comment by weslambert** [comment, marked as answer]
The source code for Security Onion can be found here:
https://github.com/Security-Onion-Solutions/securityonion

Are you looking for something specific?

**Reply by wangfei906** [reply]
I'm looking for page display code

**Reply by wangfei906** [reply]
![image](https://user-images.githubusercontent.com/37692493/176841369-f0ade4d0-4be9-4897-bf30-5b52a371a2b3.png)

**Reply by wangfei906** [reply]
It's the code of the above picture

**Reply by dougburks** [reply]
Security Onion Console source code:
https://github.com/Security-Onion-Solutions/securityonion-soc
